templates/an-cash-dao/contracts/BaseTemplate.sol

```diff
@@ -0,0 +1,359 @@
+        address _manager
+    )
+        internal
+    {
+        _acl.createPermission(ANY_ENTITY, _voting, _voting.CHALLENGE_ROLE(), _manager);
+
+        _acl.createPermission(_grantee, _voting, _voting.CHANGE_VOTE_TIME_ROLE(), _manager);
+        _acl.createPermission(_grantee, _voting, _voting.CHANGE_SUPPORT_ROLE(), _manager);
+        _acl.createPermission(_grantee, _voting, _voting.CHANGE_QUORUM_ROLE(), _manager);
+        _acl.createPermission(_grantee, _voting, _voting.CHANGE_DELEGATED_VOTING_PERIOD_ROLE(), _manager);
+        _acl.createPermission(_grantee, _voting, _voting.CHANGE_QUIET_ENDING_ROLE(), _manager);
+        _acl.createPermission(_grantee, _voting, _voting.CHANGE_EXECUTION_DELAY_ROLE(), _manager);
+    }
+
+    /* FINANCE */
+
+    function _installFinanceApp(Kernel _dao, Vault _vault, uint64 _periodDuration) internal returns (Finance) {
+        bytes memory initializeData = abi.encodeWithSelector(Finance(0).initialize.selector, _vault, _periodDuration);
+        return Finance(_installNonDefaultApp(_dao, FINANCE_APP_ID, initializeData));
+    }
+
+    function _createFinancePermissions(ACL _acl, Finance _finance, address _grantee, address _manager) internal {
+        _acl.createPermission(_grantee, _finance, _finance.EXECUTE_PAYMENTS_ROLE(), _manager);
+        _acl.createPermission(_grantee, _finance, _finance.MANAGE_PAYMENTS_ROLE(), _manager);
+    }
+
+    function _createFinanceCreatePaymentsPermission(ACL _acl, Finance _finance, address _grantee, address _manager) internal {
+        _acl.createPermission(_grantee, _finance, _finance.CREATE_PAYMENTS_ROLE(), _manager);
+    }
+
+    function _grantCreatePaymentPermission(ACL _acl, Finance _finance, address _to) internal {
+        _acl.grantPermission(_to, _finance, _finance.CREATE_PAYMENTS_ROLE());
+    }
+
+    function _transferCreatePaymentManagerFromTemplate(ACL _acl, Finance _finance, address _manager) internal {
+        _acl.setPermissionManager(_manager, _finance, _finance.CREATE_PAYMENTS_ROLE());
+    }
+
+    /* TOKEN MANAGER */
+
+    function _installTokenManagerApp(
+        Kernel _dao,
+        MiniMeToken _token,
+        bool _transferable,
+        uint256 _maxAccountTokens
+    )
+        internal returns (TokenManager)
+    {
+        TokenManager tokenManager = TokenManager(_installNonDefaultApp(_dao, TOKEN_MANAGER_APP_ID));
+        _token.changeController(tokenManager);
+        tokenManager.initialize(_token, _transferable, _maxAccountTokens);
+        return tokenManager;
+    }
+
+    function _createTokenManagerPermissions(ACL _acl, TokenManager _tokenManager, address _grantee, address _manager) internal {
+        _acl.createPermission(_grantee, _tokenManager, _tokenManager.MINT_ROLE(), _manager);
+        _acl.createPermission(_grantee, _tokenManager, _tokenManager.BURN_ROLE(), _manager);
+    }
+
+    function _mintTokens(ACL _acl, TokenManager _tokenManager, address[] memory _holders, uint256[] memory _stakes) internal {
```

Comment:
Just wondering, for the sake of simplicity, if we should remove the unused overloads?